config/deploy/webhook/deployment.yaml.tpl

```diff
@@ -41,7 +41,7 @@ spec:
               topologyKey: "kubernetes.io/hostname"
       tolerations:
         - effect: NoSchedule
-          key: node-role.kubernetes.io/infra
```

Comment:
Why were the labels in this file changed?

Reply:
Was actually intentional, but I misunderstood something: I thought it would generally be desired to keep from running on infra nodes. Reverting